### DISCUSSION TITLE
Add `skip_serializing_if` when codegen

### DISCUSSION THREAD

**Discussion by YoshieraHuang** [SeaQL/sea-orm#647] [Ideas]
`#[serde(skip_serialzing_if = "Option::is_none")]` is a very useful annotation used to skip serializing fields which values are `none`. How about considering to automatically add this annotation to fields with type `Option` in the `Model` when codegen?

**Comment by billy1624** [comment]
We could, but this has to be opt-in.

**Comment by billy1624** [comment]
Wonder what's your use case? Web API?

**Comment by YoshieraHuang** [comment]
> We could, but this has to be opt-in.

Opt-in is fine.

**Comment by YoshieraHuang** [comment]
> Wonder what's your use case? Web API?

Yes, I build my web api with sea-orm and axum.

**Comment by billy1624** [comment]
> > We could, but this has to be opt-in.
> 
> 
> 
> Opt-in is fine.

Do you want to implement it? I can guide you lolll

**Comment by YoshieraHuang** [comment]
> > > We could, but this has to be opt-in.
> > 
> > 
> > Opt-in is fine.
> 
> Do you want to implement it? I can guide you lolll

Ok, I will try to implement it.

**Comment by vrozhdestvenskyi** [comment]
Hey guys! Any updates here? :)
